polaris-react/config/jest-transform-image.js

```diff
@@ -3,6 +3,8 @@ const {basename} = require('path');
 module.exports = {
   process(src, filename) {
     const stringifedBasename = JSON.stringify(basename(filename));
-    return `module.exports = ${stringifedBasename};`;
+    return {
+      code: `module.exports = ${stringifedBasename};`,
+    };
```

Comment:
[As per the upgrade guide](https://jest-archive-august-2023.netlify.app/docs/28.x/upgrading-to-jest28/#transformer)